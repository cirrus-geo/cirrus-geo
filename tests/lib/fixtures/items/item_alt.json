{
    "type": "Feature",
    "stac_version": "1.0.0-beta.2",
    "stac_extensions": ["eo", "view", "proj"],
    "id": "S2B_17HQD_20201103_0_L2A",
    "bbox": [
      -78.85549714480591, -33.5090673116811, -77.66654434530413,
      -33.171471085391424
    ],
    "geometry": {
      "type": "Polygon",
      "coordinates": [
        [
          [-78.84720324626629, -33.5090673116811],
          [-78.85549714480591, -33.171471085391424],
          [-78.68855580607664, -33.20224951876674],
          [-78.68565102210097, -33.20281229571297],
          [-78.43490780996153, -33.25398334754271],
          [-78.12962160400237, -33.31843449933885],
          [-77.92995605454519, -33.36103653713505],
          [-77.66899441916789, -33.41891691608237],
          [-77.66654434530413, -33.482909622076725],
          [-78.84720324626629, -33.5090673116811]
        ]
      ]
    },
    "properties": {
      "start_datetime": "2020-11-03T15:22:26Z",
      "end_datetime": "2020-11-04T15:22:26Z",
      "platform": "sentinel-2b",
      "constellation": "sentinel-2",
      "instruments": ["msi"],
      "gsd": 10,
      "view:off_nadir": 0,
      "proj:epsg": 32717,
      "sentinel:utm_zone": 17,
      "sentinel:latitude_band": "H",
      "sentinel:grid_square": "QD",
      "sentinel:sequence": "0",
      "sentinel:product_id": "S2B_MSIL2A_20201103T152219_N0214_R082_T17HQD_20201103T185442",
      "sentinel:data_coverage": 20.61,
      "sentinel:valid_cloud_cover": true,
      "created": "2020-11-10T00:00:00Z",
      "updated": "2020-11-11T00:00:00Z"
    },
    "collection": "sentinel-s2-l2a",
    "assets": {},
    "links": {}
  }
