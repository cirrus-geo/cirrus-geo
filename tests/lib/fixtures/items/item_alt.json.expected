{
  "collection": {
    "DataType": "String",
    "StringValue": "sentinel-s2-l2a"
  },
  "bbox.ll_lon": {
    "DataType": "Number",
    "StringValue": "-78.85549714480591"
  },
  "bbox.ll_lat": {
    "DataType": "Number",
    "StringValue": "-33.5090673116811"
  },
  "bbox.ur_lon": {
    "DataType": "Number",
    "StringValue": "-77.66654434530413"
  },
  "bbox.ur_lat": {
    "DataType": "Number",
    "StringValue": "-33.171471085391424"
  },
  "start_datetime": {
    "DataType": "String",
    "StringValue": "2020-11-03T15:22:26Z"
  },
  "end_datetime": {
    "DataType": "String",
    "StringValue": "2020-11-04T15:22:26Z"
  },
  "status": {
    "DataType": "String",
    "StringValue": "updated"
  }
}
